style.css

```diff
@@ -142,6 +134,10 @@ body {
   color: #fff;
 }
 
+.is-hidden {
```

Comment:
moved it at the end so that this utility class is separated from the other classes.